service/server.py

```diff
@@ -25,6 +25,8 @@
 POSTCODE_REGEX = re.compile(address_utils.BASIC_POSTCODE_REGEX)
 NOF_SECS_BETWEEN_LOGINS = 1
 LOGGER = logging.getLogger(__name__)
+INCLUDE_TIME_IN_DATE_FORMAT = True
+IGNORE_TIME_IN_DATE_FORMAT = False
```

Comment:
It's me hating the thought that I send true/false to a method and would have to look at the method to find out what it does rather than guessing based on the parameters. Possibly me overthinking it and is a waste of resources creating them for a one off thing.
I'm happy to go for a straight true/false